plugins/modules/digital_ocean_load_balancer.py

```diff
@@ -591,6 +606,12 @@ def create(self):
                     ),
                 )
 
+        # Droplet ID and tag are mutually exclusive, check that both have not been defined
+        if self.module.params.get('droplet_ids') is not None and self.module.params.get('tag') is not None:
```

Comment:
Using [`mutally_exclusive`](https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#argument-spec-dependencies) for the parameter is probably a better way to go about this.

Reply:
Thank you for pointing this out